keps/sig-network/1860-kube-proxy-IP-node-binding/README.md

```diff
@@ -149,3 +159,284 @@ On downgrade, the feature gate will simply be disabled, and as long as `kube-pro
 ### Version Skew Strategy
 
 Version skew from the control plane to `kube-proxy` should be trivial since `kube-proxy` will simply ignore the `ipMode` field.
+
+## Production Readiness Review Questionnaire
+
+### Feature Enablement and Rollback
+
+###### How can this feature be enabled / disabled in a live cluster?
+
+- [x] Feature gate (also fill in values in `kep.yaml`)
+  - Feature gate name: LoadBalancerIPMode
+  - Components depending on the feature gate: kube-proxy, kube-apiserver
```

Comment:
Also cloud-controller-manager?

It will be [potentially] setting this field, right?

Reply:
Yes. I'll also add it in kep.yaml.